### PR TITLE
fix: various bugs

### PR DESCRIPTION
**Pull request by gzuuus**

### CODE REVIEW

File: packages/app/src/lib/components/passPromt.svelte [hunk 1]
```diff
@@ -13,17 +13,21 @@
...
 
+	import Spinner from './assets/spinner.svelte'
 	import Pattern from './Pattern.svelte'
 	import Checkbox from './ui/checkbox/checkbox.svelte'
 	import CAvatar from './ui/custom-components/c-avatar.svelte'
 	import Label from './ui/label/label.svelte'
 
 	export let dialogOpen = false
 	export let accointInfo: NsecAccount
+	let loading = false
```
Comment (Aslemammad) [inline packages/app/src/lib/components/passPromt.svelte:24]: i think the inconsistency between naming of "isLoading" and "loading" does not look that good!

File: packages/app/src/lib/components/product/product-item.svelte [hunk 2]
```diff
@@ -2,17 +2,18 @@
...
 	import type { DisplayProduct } from '$lib/server/products.service'
 	import * as Card from '$lib/components/ui/card/index.js'
+	import * as DropdownMenu from '$lib/components/ui/dropdown-menu'
 	import { KindProducts, KindStalls } from '$lib/constants'
 	import { createCurrencyConversionQuery } from '$lib/fetch/products.queries'
 	import { handleAddToCart } from '$lib/stores/cart'
 	import { openDrawerForProduct } from '$lib/stores/drawer-ui'
 	import ndkStore from '$lib/stores/ndk'
 	import { formatSats, parseCoordinatesString, stringToHexColor } from '$lib/utils'
+	import { Edit, MoreVertical, ShoppingCart } from 'lucide-svelte' // Added icons
```
Comment (Aslemammad) [inline packages/app/src/lib/components/product/product-item.svelte:12]: useless comment?

File: packages/app/src/lib/components/product/product-item.svelte [hunk 3]
```diff
@@ -28,77 +29,100 @@
...
-				<span style={`color:${stringToHexColor(String(name || identifier))}`} class="i-mdi-package-variant-closed w-16 h-16" />
-			</div>
-		{/if}
-	</Card.CardContent>
-	<a href={userNip05 ? `/products/${userNip05}/${identifier}` : `/products/${productCoordinates.coordinates}`}>
-		<Card.Footer class="cursor-pointer">
-			<div class="flex flex-col justify-between items-start w-full gap-2">
-				<div class="flex-grow">
-					<span class="text-sm truncate font-bold whitespace-normal">{name}</span>
+<Card.Root
```
Comment (Aslemammad) [inline packages/app/src/lib/components/product/product-item.svelte:32]: this is way cleaner now, thanks

File: packages/app/src/lib/components/stalls/stall-item.svelte [hunk 4]
```diff
@@ -2,10 +2,12 @@
 	import type { NDKEvent } from '@nostr-dev-kit/ndk'
 	import type { RichStall } from '$lib/server/stalls.service'
 	import * as Card from '$lib/components/ui/card/index'
+	import * as DropdownMenu from '$lib/components/ui/dropdown-menu'
 	import { normalizeStallData } from '$lib/nostrSubs/utils'
 	import { openDrawerForNewProductForStall, openDrawerForStall } from '$lib/stores/drawer-ui'
 	import ndkStore from '$lib/stores/ndk'
 	import { stringToHexColor, truncateString, truncateText } from '$lib/utils'
+	import { Edit, MoreVertical, Plus } from 'lucide-svelte' // Changed to MoreVertical
```
Comment (Aslemammad) [inline packages/app/src/lib/components/stalls/stall-item.svelte:10]: i think we can remove this!

**Review by Aslemammad — CHANGES_REQUESTED** [review]

File: packages/app/src/lib/components/passPromt.svelte (hunk 1 above)
Comment (gzuuus) [reply packages/app/src/lib/components/passPromt.svelte:24]: nice

**Review by Aslemammad — APPROVED** [review]

**Pull request merged**